clang/test/Driver/sycl-offload-old-model.cpp

```diff
@@ -125,8 +125,8 @@
 // RUN:    | FileCheck -check-prefix CHECK_SECTION %s
 // RUN:  %clangxx -### -target x86_64-unknown-linux-gnu %S/Inputs/SYCL/liblin64.a %s 2>&1 \
 // RUN:    | FileCheck -check-prefix NO_CHECK_SECTION %s
-// CHECK_SECTION: {{(/|\\)}}clang-offload-bundler{{.*}} "-type=ao"{{.*}} "-check-section"
-// NO_CHECK_SECTION-NOT: clang-offload-bundler{{.*}} "-type=ao"{{.*}} "-check-section"
+// CHECK_SECTION: {{(/|\\)}}clang-offload-bundler{{.*}} "-type=ao"{{.*}} "-list"
```

Comment:
should the CHECK suffix be changed here? (Nit)

Reply:
yeah, I can update the naming to match.  Also gives me a chance to refresh the bits as they have a little age to them.